modules/gitops-using-argo-cd-instance-to-manage-cluster-scoped-resources.adoc

```diff
@@ -6,6 +6,11 @@
 [id="using-argo-cd-instance-to-manage-cluster-scoped-resources_{context}"]
 = Using an Argo CD instance to manage cluster-scoped resources
 
+[WARNING]
+====
+Do not elevate the permissions of Argo CD instances to be cluster-scoped unless you have one or more rational reasons and distinct use cases. Only users with `cluster-admin` privileges must manage the instances you elevate. Anyone with access to the namespace of a cluster-scoped instance can elevate their privileges on the cluster to become a cluster administrator themselves.
```

Comment:
Can you be more specific in the first sentence? I'm sure most users feel they have "rational reasons" to do whatever they do :-).

Perhaps rewrite to something like this:

 ```suggestion
Do not elevate the permissions of Argo CD instances to be cluster-scoped unless you have a distinct use case that requires it. Only users with `cluster-admin` privileges should manage the instances you elevate. Anyone with access to the namespace of a cluster-scoped instance can elevate their privileges on the cluster to become a cluster administrator themselves.
```